composer.json

```diff
@@ -24,10 +24,10 @@
         "issues": "https://github.com/RusticiSoftware/TinCanPHP/issues"
     },
     "require": {
-        "php": "~5.4",
+        "php": "~5.5 || ^7.0.3",
         "ext-openssl": "*",
-        "namshi/jose": "~4.0",
-        "willdurand/negotiation": "~1.3"
+        "namshi/jose": "^7.2.1",
```

Comment:
The changes between 5.x, 6.x and 7.x are fairly insignificant, such that I would expect any project using TinCanPHP and the JWT lib (which probably aren't that many anyways) to be able to (and want to) upgrade fairly easily. Moving all the way to 7 eliminates our requirement to support the older deps for the life of the 1.x release.